File: datacollector/src/main/java/de/leo/fingerprint/datacollector/ui/application/DataCollectorApplication.java
package de.leo.fingerprint.datacollector.ui.application;

import android.app.Application;

import com.jakewharton.threetenabp.AndroidThreeTen;

/**
 * Created by Yunlong on 4/22/2016.
 */
public class DataCollectorApplication extends Application {

    public static boolean WIFI_NAME_ENABLED = true;
    public static boolean LOCATION_ENABLED = true;
    public static boolean ACCELEROMETER_MAGNETOMETER_GYROSCOPE_ORIENTATION_ENABLED = true;
    public static boolean ACTIVITY_ENABLED = true;
    public static boolean ENVIRONMENT_SENSOR_ENABLED = true;
    public static boolean GOOGLE_FITNESS_ENABLED = true;
    public static boolean AMBIENT_SOUND_ENABLED = true;
    public static boolean WEATHER_ENABLED = true;
    public static String BROADCAST_EVENT = "com.example.leo.datacollector";


    @Override
    public void onCreate() {
        super.onCreate();
        AndroidThreeTen.init(this);
    }
}

File: datacollector/src/main/java/de/leo/smartTrigger/datacollector/ui/application/DataCollectorApplication.kt
package de.leo.smartTrigger.datacollector.ui.application

import android.app.Application
import android.net.wifi.p2p.WifiP2pManager
import com.jakewharton.threetenabp.AndroidThreeTen


/**
 * Created by Yunlong on 4/22/2016.
 */
class DataCollectorApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        AndroidThreeTen.init(this)
    }

    companion object {

        var WIFI_NAME_ENABLED = true
        var LOCATION_ENABLED = true
        var LOCATION_NAME_ENABLED = false
        var ACCELEROMETER_MAGNETOMETER_GYROSCOPE_ORIENTATION_ENABLED = false
        var ACTIVITY_ENABLED = true
        var ENVIRONMENT_SENSOR_ENABLED = false
        var GOOGLE_FITNESS_ENABLED = false
        var SCREEN_ON_ENABLED = false
        var AMBIENT_SOUND_ENABLED = false
        var WEATHER_ENABLED = false
    }
}

